COLLABORATOR_GUIDE.md

```diff
@@ -290,15 +478,15 @@ You can find more information [in the full LTS plan](https://github.com/nodejs/l
 
 #### How does LTS work?
 
-Once a stable branch enters LTS, changes in that branch are limited to bug
+Once a Current branch enters LTS, changes in that branch are limited to bug
```

Comment:
yeah, I changed this because we no longer refer to branches as being "stable". They are either Current or LTS...